docs/usage.md

```diff
@@ -175,7 +175,7 @@ For starting from the imputation steps, the required flags are:
 - `--tools [glimpse1, quilt, stitch]`: A selection of one or more of the available imputation tools. Each imputation tool has their own set of specific flags and input files. These required files are produced by `--steps panelprep` and used as input in:
   - `--chunks chunks.csv`: A samplesheet containing chunks per chromosome. These are produced by `--steps panelprep` using `GLIMPSE1`.
   - `--posfile posfile.csv`: A samplesheet containing a TSV with the list of positions to genotype per chromosome. These are required by tools (for STITCH/GLIMPSE1). The posfile can be generated with `--steps panelprep`.
-  - `--panel panel.csv`: A samplesheet containing the post-processed VCF. This is required by GLIMPSE1. This file can be obtained with `--steps panelprep`.
+  - `--panel panel.csv`: A samplesheet containing the post-processed VCF (required by GLIMPSE1) or the hap_legend files (required by QUILT). This file can be obtained with `--steps panelprep`.
```

Comment:
The vcf form is needed also for GLIMPSE2.
We should make clear here that at least one of them is mandatory and that the hap_legend can be generated with the step panel_prep from the vcf.